week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -112,5 +139,6 @@ test(
     "18%",
     "100%",
     "0.37%"
-  ])
-);
\ No newline at end of file
+  ]
+  ))
+}
```

Comment:
There shouldn't be this brace here. It is a result of missing one earlier. Try to keep an eye on your indentation. It can help spot issues like this. It is very easy to misplaces braces and brackets when coding, use an IDE to help keep track of them. They often highlight the matching brace / bracket.